style.css

```diff
@@ -405,3 +448,229 @@ textarea::placeholder {
   place-items: center;
   margin-top: 55px;
 }
+
+@media screen and (min-width: 768px) {
+  .landing-page {
+    width: 1585px;
```

Comment:
- [ ] One of the major requirements in this project is to have a responsive design. giving our containers exact widths  like in this line makes our design unresponsive. kindly consider removing it from here and all other exact widths and heights you are giving in your containers and tags. we normally avoid them and give widths in percentages `%` or use `vw`. we also use margins and paddings. checkout this [link for more info](https://www.theodinproject.com/paths/full-stack-javascript/courses/advanced-html-and-css#responsive-design)